tutorials-v5/visualization/animation-demo.md

```diff
@@ -0,0 +1,144 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.13.8
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+# Animation demos
+For more information about QuTiP see [http://qutip.org](http://qutip.org)
+
+
+## Overview
+QuTiP has animation functions to visualize the time evolution of quantum dynamics.
+
+
+```python
+import numpy as np
+import matplotlib.pyplot as plt
+from IPython.display import HTML
+from qutip import (ket, basis, tensor, sigmaz, qeye, mesolve, anim_schmidt,
+                   complex_array_to_rgb, spin_q_function,
+                   anim_spin_distribution, about)
+```
+
+
+```python
+# a magic command enabling you to see animations in your jupyter notebook
+%matplotlib notebook
+```
+
+
+## Time evolution of a qubit
+Consider a system composed of two qubits. Its hamiltonian is $\sigma_z \otimes \mathbf{1}$ and the initial state is an entangled state ($\left|10\right>$+$\left|01\right>$)/$\sqrt2$.
+This operator acts on the first qubit and leaves the second qubit unaffected.
+
+
+```python
+# Hamiltonian
+H = tensor(sigmaz(), qeye(2))
+
+# initial state
+psi0 = (ket('10')+ket('01')).unit()
+
+# list of times for which the solver should store the state vector
+tlist = np.linspace(0, 3*np.pi, 100)
+
+results = mesolve(H, psi0, tlist, [], [])
+
+fig, ani = anim_schmidt(results)
+```
+
+
+The magic code may not work in your environments. This may happen if you run jupyter on Linux or use Google Colab. The code below will help you.
+
+
+```python
+HTML(ani.to_jshtml())
+```
+
+## Animation with plots
+You can make an animation with plots. Note that you cannot have it with other animations.
```

Comment:
Just a small suggestion for clarity:
"You can make an animation with accompanying plots. Note that you cannot have it with additional accompanying animations."